sda/cmd/download/swagger_v1.yml

```diff
@@ -0,0 +1,241 @@
+openapi: 3.0.3
+info:
+  title: SDA file download API
+  version: "1.0"
```

Comment:
```suggestion
  version: "1.0"
  description: |
     The SDA file download API enables querying of dataset content metadata and content retrieval.
     Note: unless otherwise specified, whenever coordinates are used in this specification, they reference the
     coordinate space of the re-encrypted header followed by the archive content.  
```

?